betting-game-react/src/components/AboutUs/AboutUs.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const AboutUs: React.FC = () => (
```

Comment:
So we talked about me not using the `: React.FC` right? 
Well, turns out that at least IDEA does not understand the type of the components this way, so I actually prefer your way :)